lib/options.js

```diff
@@ -306,6 +307,13 @@ module.exports = function(usingFlatConfig) {
                 default: void 0,
                 description: "Adds reported errors for unused eslint-disable and eslint-enable directives"
             },
+            {
+                option: "report-unused-disable-directives-severity",
+                type: "String",
+                default: void 0,
+                description: "Chooses severity level for reporting unused eslint-disable directives",
```

Comment:
```suggestion
                description: "Chooses severity level for reporting unused eslint-disable and eslint-enable directives",
```

Since the description for `report-unused-disable-directives` mentions `eslint-enable` directives, not mentioning them in the description for `report-unused-disable-directives-severity` might imply that this option does not apply to them.